src/main/webapp/WEB-INF/views/admin/edit-room.jsp

```diff
@@ -100,6 +98,52 @@
         </tr>
     </table>
 </div>
+<div id="deactivateModal" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-body text-center">
+                <br>
+                <h2><spring:message code="room.deactivate"/></h2>
+                <p class="cancelName ">
+                    <spring:message code="booking.confirmCancelQuestion2"/>
+                </p>
+                <button id="deactivateYesButton" class="btn  btn-success">
+                    <spring:message code="room.yes_button" />
+                </button>
+                <button id="deactivateNoButton" class="btn btn-danger" data-dismiss="modal">
+                    <spring:message code="room.no_button"/>
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
+<div id="activateModal" class="modal fade">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-body text-center">
+                <br>
+                <h2><spring:message code="room.activate"/></h2>
+                <button id="activateYesButton" class="btn  btn-success">
+                    <spring:message code="room.ok_button" />
```

Comment:
There is no confirmation message because it only need to tell the admin that the operation procced correctly